lib/pbench/server/indexer.py

```diff
@@ -15,6 +15,7 @@
 import sys
 import tarfile
 import errno
+import pathlib
```

Comment:
We can import "from pathlib import Path" this way we can avoid using "pathlib.Path" in code and we can directly use just "Path"